enhancements/installer/inputs-from-stdin.md

```diff
@@ -0,0 +1,182 @@
+
+```
+TMPDIR=$(mktemp -d)
+kustomize build https://github.com/myorg/myblueprint.git//install-config/ | openshift-install --kustomization create manifests -f - --dir=$TMPDIR
+cat <<- EOF >> $TMPDIR/kustomization.yaml
+components:                                                                                                                                                                                                     
+- git::https://github.com/myorg/myblueprint.git//manifest-overlay/
+EOF
+kustomize build $TMPDIR | openshift-install create cluster -f -
+```
+
+The Kustomize `Component` can contain a set of patches that will overwrite certain parameters, or even add new manifests, resources, etc. To show one example, let's say we want to patch etcd service port. The files located in the remote Git repository will contain:
+
+**kustomization.yaml**
+
+```
+apiVersion: kustomize.config.k8s.io/v1alpha1  # <-- Component notation
+kind: Component
+
+patches:
+- etcd-service-port.yaml
+- etcd-host-service-port.yaml
+```
+
+**etcd-service-port.yaml**
+```
+apiVersion: v1
+kind: Service
+metadata:
+  name: etcd
+  namespace: openshift-etcd
+  labels:
+    k8s-app: etcd
+spec:
+  ports:
+    - $patch: replace
+    - name: etcd
+      port: 2378
+      protocol: TCP
+```
+
+**etcd-host-service-port.yaml**
+```
+apiVersion: v1
+kind: Service
+metadata:
+  name: host-etcd
+  namespace: openshift-etcd
+  labels:
+    k8s-app: etcd
+spec:
+  ports:
+    - $patch: replace
+    - name: etcd
+      port: 2378
+      protocol: TCP
+```
+
+
+### User Stories
+
+#### Story 1: Administrator persona
+
+As an administrator, I want to deploy a set of nearly identical clusters by overlaying minimal changes to a base cluster definition using a simple pipelined CLI command. If I have to apply the same specific configuration on all my clusters, I need an easy way to apply it and deploy. I want to use kustomize or any other manifest manipulation tool and pipe the output to the OpenShift Installer. 
+
+#### Story 2: Developer persona
+
+As a developer, I want to customize the definition of a production cluster in order to deploy a copy in my test environment. In order to facilitate this task, It would help to have available a `kustomization.yaml` with a list of the manifest files generated by the installer.
+
+### Implementation Details/Notes/Constraints
+
+The installer would need to keying assets based on their type and name rather than their on-disk path. This approach does not fundamentally alter the design of the installer. It must also apply the existing validations on all of the assets as if they were provided via the filesystem.
+
+#### Resources created by the installer
+
+The `openshift-install create manifests` will create a kustomization.yaml file that will list the resources located in the openshift and the manifest directories if the flag `--kustomize` is passed. The content of the file should look like this:
```

Comment:
I would rather not have the installer own code around supporting a kustomize.yaml file, particularly when the list of resources is simply all of the files.
```
(echo "resources:"; find "${TMPDIR}" -type f -not -name '\.*' -printf '- %P\n') > "${TMPDIR}/kustomization.yaml"
```